apps/demos/Demos/Chat/Overview/React/App.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+import { Chat, ChatTypes } from 'devextreme-react/chat';
+
+export default function App() {
+    const date = new Date();
```

Comment:
Kindly check the other demo files and let's use the same indentation as in those. I am not sure about all the files, but for App.tsx, we usually use a 2-space indent.